codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttMessageBuilders.java

```diff
@@ -536,7 +536,7 @@ public PubAckBuilder reasonCode(byte reasonCode) {
             return this;
         }
 
-        public PubAckBuilder packetId(short packetId) {
+        public PubAckBuilder packetId(int packetId) {
```

Comment:
I think we need to keep the old as well as otherwise it will be a breaking change.